src/api.ts

```diff
@@ -0,0 +1,39 @@
+import { config } from "./config"
```

Comment:
@huseyincansoylu this makes sense.
the only thing is that you made this file in this PR but you're not using it.
what I suggest is:
you remove this file from this PR
I can open a new issue to make this file and use it all accross the application

Reply:
also I think this may be done in the github context

Reply:
okay i can do that